pages/apis/graphql/cookbooks/clusters.md

```diff
@@ -184,3 +184,21 @@ query getClusterQueueAgent {
 }
 ```
 
+## Associate a pipeline with a cluster
+
+First, [get the Cluster ID](#get-a-pipelines-id) you want to associate the Pipeline with.
```

Comment:
Thanks again @tomowatt !

Looks like what you're linking to is on a different page...

```suggestion
First, [get the Cluster ID](/docs/apis/graphql/cookbooks/pipelines#get-a-pipelines-id) you want to associate the Pipeline with.
```

Reply:
Cheers for spotting @gilesgas the error threw me off when looking into the Build due to the matching header names, my bad, but sorted now!